staging/src/k8s.io/apiserver/pkg/endpoints/handlers/create.go

```diff
@@ -113,6 +116,31 @@ func createHandler(r rest.NamedCreater, scope RequestScope, admit admission.Inte
 		// TODO: replace with content type negotiation?
 		includeUninitialized := req.URL.Query().Get("includeUninitialized") == "1"
 
+		newIntent := make(map[string]interface{})
```

Comment:
You should maybe create a helper function (with some tests) that will be used by `apply.go` and `create.go`.

Reply:
Sounds good, where do helpers like this usually go?

Reply:
Since it's only used in this package, I'd just create another file in the same package.